Write "looking ahead" sections for all chapters (#511)
worldbank/dime-data-handbook: chapters/6-analysis.tex

so papers can be more easily be formatted into their specific layout.

\section{Looking ahead}

> This section discussed the steps needed to take original data
> and create analysis datasets and outputs from it.

Reviewer: ```suggestion
This chapter discussed the steps needed to 
create analysis datasets and outputs from original data.
```

This process created new derived information
from measurements taken in the field or collected by partners,
and combined these measures with the research design

> described in the data map and the analysis plan.

Reviewer: how can information be 'combined' with the research design. perharps organize it for analysis according to the research design, as outlined in the Data Map?

Author: Same as above -- this is meant to refer to merging with master datasets according to the other plans, so literally to be merging research variables and measurement variables as we've defined. How to better express that?

Reviewer: How about something along the lines of: By combining the variables of interest for the analysis (measurement variables) with the variables describing the analytical design (research variables), you created original datasets ready for analysis.

Author: ```suggestion
By combining the observed variables of interest for the analysis (measurement variables) 
with the information in the data map describing the study design (research variables), 
you created original datasets ready for analysis.
```

Doing so is difficult, creative work,
and it is impossible to be reproduced by another person
without access to detailed records and explanations
of how you interpreted and modified the data.
Creating organized and well-documented code
for both the creation of new information
and the creation of research outputs describing
patterns you identified in the data --
and used to answer your research questions --
provides this knowledge to both future users of the original data
as well as readers interested in completely understanding

> the results you obtained.

Reviewer: Overly complex sentence. perhaps:
You learned that code must be well-organized and well-documented to allow others to understand how research outputs were created and how those were used to answer the research questions. 
?

Author: ```suggestion
You learned that code must be well-organized and well-documented 
to allow others to understand how research outputs were created 
and how those were used to answer the research questions.
```

The final chapter of this book will provide a guide

> to pulling these raw findings together into publishable work,

Reviewer: ```suggestion
to assembling the raw findings into publishable work,
```
